src/git_when_merged.py

```diff
@@ -307,7 +307,7 @@ def __init__(self, refname, parents):
         self.msg = 'Merged via multiple parents: %s' % (' '.join(parents),)
 
 
-def find_merge(commit, branch):
+def find_merge(commit_sha1, branch):
```

Comment:
FWIW, this could theoretically break downstream code that relies on this name. For example:
``` python
import git_when_merged
git_when_merged.find_merge(commit="123abcd")
```
This is the error that would manifest after this change:
```
TypeError: find_merge() got an unexpected keyword argument 'commit'
```

I suspect the risk of that is low but figured it was worth calling out.

One way to avoid that is to specify that callers MUST use a positional argument:
```suggestion
def find_merge(commit_sha1, /, branch):
```
- Note: This is a 3.8 feature. It would cause a SyntaxError in 3.7 or lower (but all those versions are EOL now).

Reply:
Thanks for the review and for pointing this out. While I agree that this is theoretically a problem, I think that the improved clarity (especially if somebody is invoking this directly!) is worth the tiny risk of backwards-incompatibility at the API level.